src/accel/TrackingAction.hh

```diff
@@ -18,9 +21,20 @@ namespace celeritas
 class TrackingAction final : public G4UserTrackingAction
 {
   public:
-    TrackingAction();
+    //!@{
+    //! \name Type aliases
+    using SPParams      = std::shared_ptr<SharedParams>;
```

Comment:
Since the params are going to be initialized at this point and this class won't change the pointed-to data, we could make this an `shared_ptr<const SharedParams>`.